app/src/components/main/grid/gridBody/nestedGridRow/nestedGridRow.jsx

```diff
@@ -108,10 +108,38 @@ export const NestedGridRow = ({
   const showPreviousButton = isLoadPreviousButtonVisible(page);
   const showMoreButton = isLoadMoreButtonVisible(page);
   const showLoadCurrentStepButton = isLoadCurrentStepButtonVisible(page, data, allNestedStepsData);
+  const [firstRowId, setFirstRow] = React.useState(null);
+  const [direction, setDirection] = React.useState(null);
```

Comment:
Maybe, destructure useState and useRef from react library?

Reply:
In this case, I think, that unnecessary to increase the size of imported module.
We have everything in React namespace.